changelog/april2024/2024-04-04-cockpit-added-metrics-degrissivity.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: Metrics degrissivity
+status: added
+author:
+    fullname: 'Join the #[xxx] channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2024-04-04
+category: observability
+product: cockpit
+---
+
+We are pleased to announce degressive pricing on Cockpit external metrics.
+
+We add 6 levels for degressive monthly pricing : 
+- 0 to 10B metrics : 0.15€/ Million samples
+- 10 to 25B metrics : 0.13€/ Million samples
+- 25B to 50B metrics : 0.12€/ Million samples
+- 50 to 100B metrics : 0.11€/ Million samples
+- 100 to 200B metrics : 0.10€/ Million samples
+- 200B and more : 0.08€/ Million samples
+
+Pricing is applied **in its level**. So if you send 52 Billion external samples a month, you'll be billed as so :
+- 10000*0,15 : 1500€
+- 15000*0,13 : 1950€
+- 25000*0,12 : 3000€
+- 2000*0,11 : 220€
+
+Total : 6670€ instead of 7800€
```

Comment:
```suggestion
Certainly! Here's an improved version of the changelog:

---
We're thrilled to unveil our latest update: degressive pricing for Cockpit external metrics. 
Now, you can enjoy even more cost-efficient monitoring solutions tailored to your needs.

**Six levels of degressive monthly pricing:**

- 0 to 10 billion metrics: €0.15 per Million samples
- 10 to 25 billion metrics: €0.13 per Million samples
- 25 to 50 billion metrics: €0.12 per Million samples
- 50 to 100 billion metrics: €0.11 per Million samples
- 100 to 200 billion metrics: €0.10 per Million samples
- 200 billion and beyond: €0.08 per million samples

**How it works:**

Our pricing is applied at each level, ensuring you get the best value for your usage. For instance, if you send 52 billion external samples in a month, here is how you will be billed:

- 10,000 * €0.15: €1,500
- 15,000 * €0.13: €1,950
- 25,000 * €0.12: €3,000
- 20,000 * €0.11: €2,200

**Total:** €6,650 instead of the standard €7,800.

Upgrade your monitoring experience with Cockpit and enjoy smarter pricing that scales with your needs!
```